.github/workflows/pr_checks.yml

```
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: PR Checks
+
+on: pull_request
```

The `cordova-eslint` uses this instead:

```suggestion
on: [push, pull_request]
```

The more I think about it, we might want to include `push`. In our release process, there are still a few commits that are pushed directly to the branch that are not from a PR. For example the `Update version & RELEASENOTES.md for #.#.#`.

---

The reason I included `push` is also to replicate the current CI behavior. So that you can easily trigger CI runs in your fork just by pushing a branch.